TLM/TLM/UI/SubTools/LaneArrows/LaneArrowToolWindow.cs

```diff
@@ -87,13 +88,13 @@ public class LaneArrowToolWindow : U.Panel.BaseUWindowPanel {
         public void SetupControls(UiBuilder<LaneArrowToolWindow> builder, int numLanes) {
             Buttons = new List<LaneArrowButton>();
 
-
             using (var buttonRowBuilder = builder.ChildPanel<U.Panel.UPanel>(
                 setupFn: p => { p.name = "TMPE_ButtonRow"; })) {
                 // buttonRowBuilder.SetPadding(Constants.U_UI_PADDING);
```

Comment:
commented out code without TODO (although this one is not your fault)

Reply:
It is leftover from previous PR when i was figuring out correct paddings.